pandas/core/internals.py

```diff
@@ -1108,7 +1108,8 @@ def check_int_bool(self, inplace):
         # a fill na type method
         try:
             m = missing.clean_fill_method(method)
-        except:
+        except Exception:
```

Comment:
the entire point of correctly lining is to catch a more specific error
don’t put Execption everywhere with TODO that is not useful
his is why we are not raising on master atm with the latest linter yet

Reply:
Do you know the appropriate exceptions to catch? I was going to fix these but didn't know what could be raised, and didn't want to break things.

Reply:
you simply put a breakpoint in the exception and run the tests

Reply:
Well looking at clean_fill_method the only thing it'll raise is ValueError, so this one I can make specific.

Reply:
Ditto for clean_interp_method.